src/main/java/com/expediagroup/hiveberg/IcebergInputFormat.java

```diff
@@ -67,30 +71,34 @@
     }
     HadoopCatalog catalog = new HadoopCatalog(job,location.getPath());
     TableIdentifier id = TableIdentifier.parse(job.get(TABLE_NAME));
-    catalog.loadTable(id);
     table = catalog.loadTable(id);
 
+    //String[] readColumns = ColumnProjectionUtils.getReadColumnNames(job);
```

Comment:
So this needs to go? Does this mean we read all columns from the file instead of just a subset?

Reply:
This was what I was originally trying to get working, but i'm still working on it so i'll open another PR when i make some progress